File: app/src/main/java/org/metabrainz/mobile/data/sources/api/entities/response/RecordingSearchResponse.java
package org.metabrainz.mobile.data.sources.api.entities.response;

import org.metabrainz.mobile.data.sources.api.entities.mbentity.Recording;

import java.util.List;

public class RecordingSearchResponse extends SearchResponse {

    private List<Recording> recordings;

    public List<Recording> getRecordings() {
        return recordings;
    }

    public void setRecordings(List<Recording> recordings) {
        this.recordings = recordings;
    }
}

File: app/src/main/java/org/metabrainz/android/data/sources/api/entities/response/RecordingSearchResponse.kt
package org.metabrainz.android.data.sources.api.entities.response

import org.metabrainz.android.data.sources.api.entities.mbentity.MBEntity
import org.metabrainz.android.data.sources.api.entities.mbentity.Recording

class RecordingSearchResponse : SearchResponse<MBEntity?>() {
    var recordings: List<Recording>? = null
}